### PR TITLE
1.02 update

### PR DESCRIPTION
**Pull request by CharlieMacnamara**
- Added parameter description for enum AreaCovered 'all' value.
- Added note for consumption records.
- Fixed full-stop inconsistencies within changelog.

### CODE REVIEW

File: source/includes/_consumption.md [hunk 1]
```diff
@@ -233,6 +233,10 @@ Consumption records which failed to upload because the associated meter or asset
 **Other Errors**  
 Consumption records which were rejected along with the reason for the rejection.
 
+<aside class="notice">
+SEIRA always specifies consumption amounts in <code>kWh</code> for all energy and <code>m3</code> for water. Once in the system, no conversion is performed on these values because all aggregations are calculated using the values provided.
```
Comment (mattjmatthias) [inline source/includes/_consumption.md:237]: SEIRA -> SIERA

**Review by mattjmatthias — CHANGES_REQUESTED** [review]

File: source/includes/_consumption.md (hunk 1 above)
Comment (CharlieMacnamara) [reply source/includes/_consumption.md]: 🤦

File: source/includes/_consumption.md [hunk 2]
```diff
@@ -233,6 +233,10 @@ Consumption records which failed to upload because the associated meter or asset
 **Other Errors**  
 Consumption records which were rejected along with the reason for the rejection.
 
+<aside class="notice">
+SIERA always specifies consumption amounts in <code>kWh</code> for all energy and <code>m3</code> for water. Once in the system, no conversion is performed on these values because all aggregations are calculated using the values provided.
```
Comment (mattjmatthias) [inline source/includes/_consumption.md:237]: m<sup>3</sup> for the m3, sorry forgot to add this last time

**Review by mattjmatthias — CHANGES_REQUESTED** [review]

File: source/includes/_consumption.md (hunk 2 above)
Comment (CharlieMacnamara) [reply source/includes/_consumption.md:237]: No worries. Was my assumption about formatting for the units correct?

Comment (CharlieMacnamara) [reply source/includes/_consumption.md:237]: Okay, thanks for comparing.

**Review by mattjmatthias — APPROVED** [review]

**Pull request merged**